types/app.d.ts

```diff
@@ -45,3 +45,5 @@ export function entity<T = unknown>(functionName: string, options: EntityOptions
  * @param options the configuration options for this activity, specifying the handler and the inputs and outputs
  */
 export function activity(functionName: string, options: ActivityOptions): void;
+
+export * as client from "./app.client";
```

Comment:
Yeah these are all valid options. I'm not sure we had extensive discussion on how exactly to name/organize these. I just went with one suggestion here, but I also have no strong preferences.

Reply:
Perhaps `df.app.client.http` doesn't feel right to me because I'm used to the templates calling these starter functions. Even in your example above, the function name is "httpStart" or "timerStart" without "client" in the name. Should we do something like `df.app.start.http` or `df.app.httpStart` instead? Then again I see the docs categorize these as [client functions](https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview#client-functions) so I'm a little confused here.

I don't have a strong preference, but I don't think this should be an arbitrary decision. All I ask is that you give a reason for your choice. Intellisense is a priority for the new model and the organization here directly impacts that experience.

Reply:
Okay, makes sense. Here are a few reasons why I made this decision:

* I think client functions should be under their own namespace. I don't want a bunch of client functions polluting the intellisense for `df.app`, especially if we end up adding similar methods for all possible triggers, when at the end of the day, the "meat" of the durable app is in your orchestrations, activities, entities.
* As you said, the docs consistently refer to these functions as client functions. Sometimes we also call them client starter functions, but I think client function is a more ubiquitous name. I could also see this being called `df.app.starter` for example, but I think `client` is the more "official" name for them.
* We already have precedent for this name. To register a client input binding, we call `df.input.durableClient()`. It was a narrow decision for me to choose `client` over `durableClient` for the client functions, but I eventually went for `client` because it felt like `df.app.durableClient.http()` would be a bit too lengthy, and if you're using `df.app`, it's already assumed to be a durable client. I could also easily see this being `df.app.durableClient` though.

In conclusion, I think this should be a separate namespace. `starter` or `durableClient` are other proposals for what to call that namespace, that I don't have strong feelings against, but I went with `client` for the reasons mentioned above.